irohad/consensus/yac/impl/yac_gate_impl.cpp

```diff
@@ -86,6 +87,8 @@ namespace iroha {
                 const auto model_hash =
                     hash_provider_->toModelHash(hash.value());
                 // iterate over peers who voted for the committed block
+                // TODO [IR-1753] Akvinikym 11.10.18: add exponential backoff
+                // for each peer iteration
                 rxcpp::observable<>::iterate(commit_message.votes)
```

Comment:
Seems, that here we have to shuffle vote hashes for DOS prevention. But it could be done in the separate task.

Reply:
@Akvinikym please respond before merge. Do we have the issue mentioned by Fyodor in Jira?

Reply:
Yes, absolutely, I have updated the 1753 task with this thing